main.js

```diff
@@ -22,7 +22,7 @@ function createWindow () {
   }))
 
   // Open the DevTools.
-  // win.webContents.openDevTools()
+  win.webContents.openDevTools()
```

Comment:
This line automatically opens the Electron dev tools which is handy for debugging (It's just Chrome tools after all